apps/settings/js/panels/app_storage/panel.js

```diff
@@ -53,7 +53,7 @@ define(function(require) {
         AppStorage.storage.observe('usedSize', _updateUsedSize);
         AppStorage.storage.observe('freeSize', _updateFreeSize);
 
-        _updateElements();
+        AppStorage.updateInfo();
```

Comment:
Forcing `AppStorage` update the info only makes the values stored in `AppStorage` get refreshed. We still need to call to `_updateElements` to render the values to the elements.